src/xrCore/Math/MathUtil.cpp

```diff
@@ -48,9 +50,14 @@ void Initialize()
     Skin2W = Skin2W_CPP;
     Skin3W = Skin3W_CPP;
     Skin4W = Skin4W_CPP;
-    PLCCalc = PLCCalc_SSE;
+#if defined(XR_ARM64)
+	PLCCalc = PLCCalc_CPP;
```

Comment:
Replace tabs with 4 space, please. And in else section.
```suggestion
    PLCCalc = PLCCalc_CPP;